Week3/homework/step2-2.js

```diff
@@ -4,17 +4,27 @@ function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
 
   // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  // console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);'
+  for (let i = startIndex; i <= stopIndex; i++) {
+    numbers.push(i);
+    if (i % 3 === 0) {
+      threeCallback(i);
+    }
+    if (i % 5 === 0) {
+      fiveCallback(i);
```

Comment:
Ah. What about the requirement of printing when number is divisible by both 3 and 5? I think you missed it here. Hint: a number divisible by 3 and 5 will always be divisible by 15 😉